nodeup/pkg/bootstrap/install.go

```diff
@@ -111,16 +111,19 @@ func (i *Installation) buildSystemdJob() *nodetasks.Service {
 	manifest.Set("Unit", "Documentation", "https://github.com/kubernetes/kops")
 
 	// TODO temporary code, till vsphere cloud provider gets its own VFS implementation.
```

Comment:
Nit: this is no longer a temporary code, as we are going to still with minio for a while. Please update the comment accordingly.